deploy/klusterlet/config/manifests/bases/klusterlet.clusterserviceversion.yaml

```diff
@@ -23,18 +23,18 @@ spec:
       name: klusterlets.operator.open-cluster-management.io
       version: v1
   description: |
-    The Klusterlet proivdes the registation to the Hub clusters as a managed cluster. This operator supports the installation and upgrade of Klusterlet.
+    The Klusterlet provides the registration to the Hub cluster as a managed cluster. This operator supports the installation and upgrade of Klusterlet.
 
     ## Prerequisites
-    You need a Hub cluster which has installed ClusterManager.
+    You need a Hub cluster with ClusterManager installed.
 
     ## How to Install
     * Install Klusterlet operator by following instructions in top right button `Install`.
```

Comment:
At least in the OperatorHub in OpenShift 4.14, the "Install" button doesn't appear to be in the top _right_. I'm not sure if it is in other environments. If so, maybe this could/should be a little more accommodating of potential UI changes?

Reply:
I think the install button is displayed on the https://operatorhub.io/operator/klusterlet page.

Reply:
Ah right, I didn't think of that page.

This text also appears in the in-cluster OperatorHub in OpenShift, where the "Install" button tends to be further left than the text, so I think my point about not being too specific to one particular UI may still be relevant? I don't intend to make any changes around that in this PR, just highlighting it in case it could be improved.

![image](https://github.com/open-cluster-management-io/ocm/assets/442386/4ededc46-4d89-45bc-afe4-11b8631e2afa)